src/main/java/ru/javawebinar/topjava/web/MealServlet.java

```diff
@@ -0,0 +1,84 @@
+package ru.javawebinar.topjava.web;
+
+import ru.javawebinar.topjava.dao.MealDaoImplMem;
+import ru.javawebinar.topjava.model.Meal;
+import ru.javawebinar.topjava.model.MealWithExceed;
+import ru.javawebinar.topjava.service.MealService;
+import ru.javawebinar.topjava.service.MealServiceImplMem;
+import ru.javawebinar.topjava.util.MealsUtil;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import org.slf4j.Logger;
+
+import static org.slf4j.LoggerFactory.getLogger;
+
+public class MealServlet extends HttpServlet {
+    private static final Logger LOG = getLogger(MealServlet.class);
```

Comment:
Your code should looks accurate. Da you undersatnd what I mean ?

Reply:
You mean I should delete empty strings?

Reply:
Correct